.pre-commit-config.yaml

```diff
@@ -0,0 +1,10 @@
+repos:
+-   repo: https://github.com/psf/black
+    rev: 22.8.0
+    hooks:
+    -   id: black
+-   repo: https://github.com/pycqa/flake8
+    rev: 4.0.1
+    hooks:
+    -   id: flake8
```

Comment:
Have you met problems in the past using `flake8` and `black` together?

This configuration should allow for interoperability between those two tools.

Reply:
I think with all the exclusions in the other file, they will co-exist, but I kinda wonder if there's any value left in having `flake8`?

Reply:
black mainly formats code while flake8 takes care of style conventions that black ignores. 

Since black is called before flake8, most of reformatting will be done by black, but it is possible that there are still issues (e.g. non-compliance with [PEP8](https://peps.python.org/pep-0008/)) that can remain.

We can remove flake8 if you want.